File: Backend/WebAPI/Requests/FilmRequest.cs
namespace WebAPI.Requests
{
    public class FilmRequest
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public int ReleaseYear { get; set; }
        public int DurationInMinutes { get; set; }
        public string? PosterFileName { get; set; }
    }
}

File: Frontend/src/app/Requests/FilmRequest.ts
export class FilmRequest {
    name : string;
    description : string;
    durationInMinutes : number;
    releaseYear : number;

    constructor(
        name : string,
        description : string,
        durationInMinutes : number,
        releaseYear : number
    ) {
        this.name = name;
        this.description = description;
        this.durationInMinutes = durationInMinutes;
        this.releaseYear = releaseYear;
    }
}